images/builder/docker/sti-image-builder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:centos7
+
+RUN yum install -y --enablerepo=centosplus epel-release gettext tar automake \
+      make git docker
+
+# TODO: STI should be installed from official release page, instead of bundling
+#       it in the GIT repository.
+ADD bin/sti.gz /usr/bin/sti.gz
+RUN gunzip /usr/bin/sti.gz
```

Comment:
Since the vendored STI does provide the `cmd/sti` directory I can't build the STI executable from it. Since this card is is planned for next sprint, we can defer merging this till the STI release process is in place and I can download the STI binary from official location. @bparees ?